coordinator/src/services/translation/response/tracker/translation.ts

```diff
@@ -171,20 +228,22 @@ function convertLineItemToInput(lineItemId: string, prescription: DetailPrescrip
   }
 
   const dispensingInformationExtension = []
-  if (prescription.prescriptionDispensedDate && prescription.prescriptionDispensedDate !== "False") {
+  if ("prescriptionDispensedDate" in prescription
```

Comment:
does 231 and 232 overlap?

Reply:
I don't think so. Typescript won't let you write `prescription.prescriptionDispensedDate` if `prescriptionDispensedDate` might not be in `prescription`